builder-nodejs/Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM jenkinsxio/builder-base:1
 RUN curl -f --silent --location https://rpm.nodesource.com/setup_9.x | bash - && \
```

Comment:
`https://rpm.nodesource.com/setup_9.x` should be replaced with `https://rpm.nodesource.com/setup_11.x`

Reply:
👍